src/extension/module.js

```diff
@@ -1814,6 +1870,10 @@ import sampleRUM from './rum.js';
         lines.push(...failed.map((item) => {
           if (item.status === 404) {
             item.error = getBulkText([1], 'result', operation, 'error_no_source');
+          } else if (item.status === 400) {
+            item.error = getBulkText([1], 'result', operation, 'error_too_large');
+          } else if (item.status === 415) {
+            item.error = getBulkText([1], 'result', operation, 'error_not_supported');
```

Comment:
This is unrelated to the original issue of throttling, but it helps improve error handling.

Reply:
You are right, we should better distinguish 400s.

Reply:
Done, please re-check @dylandepass